src/app/services/exchanges/exchange_service_base.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABC, abstractmethod
+
+from app.models import Currency, CurrencyPair, Exchange
+from sqlalchemy.orm import Session
```

Comment:
Separating the application local imports

```suggestion
from sqlalchemy.orm import Session

from app.models import Currency, CurrencyPair, Exchange
```